api/cmd/portainer/main_test.go

```diff
@@ -56,3 +56,59 @@ func Test_enableFeaturesFromFlags(t *testing.T) {
 		}
 	})
 }
+
+const FeatTest portainer.Feature = "optional-test"
+
+func optionalFunc(dataStore portainer.DataStore) string {
```

Comment:
```suggestion
func optionalFunc(dataStore portainer.DataStore) bool {
```

Reply:
yeah, so I wanted it to be much more explicit on failure that we're not just returning the same thing as is stored _in_ the feature flag

otherwise, its too simple to just make 

```
func optionalFunc() bool {
  return settings.FeatureFlagSettings[FeatTest]
}
```

and the idea is to show a trivial use, not just "is the feature flag true"